Engine/ac/game.cpp

```diff
@@ -2443,177 +2323,134 @@ int restore_game_data (Stream *in, const char *nametouse, SavedGameVersion svg_v
         cachedQueuedMusic = load_music_from_disk(play.music_queue[0], 0);
     }
 
-    return 0;
+    return kSvgErr_NoError;
 }
 
-int restore_game_data (Common::Stream *in, const char *nametouse)
+bool restore_game_data(Common::Stream *in, const char *nametouse)
```

Comment:
Are we converting all methods from errorcodes to bools?  Bools don't really let you have more than one error state :)